_posts/2022-11-10-git-tips-upstream-repositories-merge-conflicts-vlog.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Vlog: Git Tips on Upstream Repositories and Merge Conflicts'
+date: 2022-11-10
```

Comment:
@cam-rod Thnks!

Can you change the file name and this date to today (2022-11-11). That will bring it to the top of the news list; right now it renders in the list buried below a bunch of stuff that we added yesterday. If you do, please ping me in chat once you've pushed.

Yesterday is ok too as that is when the vlog came out, so at some point today unless you say otherwise I'll go ahead and merge this as is.